README.md

```diff
@@ -365,6 +366,34 @@ The maxLength attribute added to the input. Specifies the maximum number of char
     ...>
 ```
 
+<a name="inline"></a>
+##### inline (optional)
+The inline attribute use to determine that selected tags and input field will be put in the same div block or not. If you want to separate it and apply auto scroll for just selected tags only, change this attribute. Specifies the inline attribute.
```

Comment:
Can you change the text to  `The inline attributes decides whether the input fields and selected tags will be rendered in-line` ?